using-mobile-extensions/adobe-target/target-api-reference.md

```diff
@@ -204,173 +281,331 @@ ACPTarget.getTntId({tntId in
+withTargetParameters: (nullable ACPTargetParameters*) targetParameters;
 ```
 
+* _mboxNames_ is a NSArray of mbox locations for which display notification is to be sent to Target.
+* _targetParameters_ is the configured `ACPTargetParameters` for the request.
+
 #### Examples
 
 Here are some examples in Objective-C and Swift:
 
 **Objective-C**
 
-```text
-[ACPTarget setPreviewRestartDeepLink:@"myapp://HomePage"];
+```objectivec
+ACPTargetProduct *product = [ACPTargetProduct targetProductWithId:@"24D334" categoryId:@"Stationary"];
+
+ACPTargetOrder *order = [ACPTargetOrder targetOrderWithId:@"ADCKKBC" total:@(400.50) purchasedProductIds:@[@"34", @"125"]];
+
+ACPTargetParameters *targetParameters = [ACPTargetParameters targetParametersWithParameters:nil
+profileParameters:nil
+product:product
+order:order];
+
+[ACPTarget locationsDisplayed:@[@"mboxName1", @"mboxName2"] withTargetParameters:targetParameters];
 ```
 
 **Swift**
 
 ```swift
-ACPTarget.setPreviewRestartDeepLink("myapp://HomePage")
+let product = ACPTargetProduct(id: "24D334", categoryId: "Stationary")
+
+let order = ACPTargetOrder(id: "ADCKKBC", total: NSNumber(value: 400.50), purchasedProductIds: ["34", "125"])
+
+let targetParameters = ACPTargetParameters(parameters: nil, profileParameters: nil, product: product, order: order)
+
+ACPTarget.locationsDisplayed(["mboxName1", "mboxName2"], with: targetParameters)
 ```
 {% endtab %}
 
 {% tab title="React Native" %}
-### setPreviewRestartDeeplink
 
-**JavaScript**
+#### Syntax
 
 ```javascript
-ACPTarget.setPreviewRestartDeeplink("myapp://HomePage");
+locationsDisplayed(mboxNames: Array<string>, parameters?: ACPTargetParameters)
 ```
-{% endtab %}
-{% endtabs %}
-
-## Prefetch offers
-
-The SDK can minimize the number of times it reaches out to Target servers to fetch offers by caching server responses. With a successful prefetch call for mbox locations, offer content is retrieved and cached in the SDK. This content is retrieved from the cache for all future [retrieveLocationContent](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference#retrieve-location-content-requests) API calls for the specified mbox names. This prefetch process reduces offer load time, network calls made to Target servers, and allows Target to be notified which mbox was visited by the mobile app user.
 
-{% hint style="warning" %}
-Prefetched offer content does not persist across application launches. The prefetch content is cached as long as the application lives in memory or until the API to clear the cache is called. For more information, see [Clear prefetch offer cache](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target#clear-prefetch-offer-cache).
-{% endhint %}
+* _mboxNames_ is an Array of mbox locations for which display notification is to be sent to Target.
+* _targetParameters_ is the configured `ACPTargetParameters` for the request.
```

Comment:
Please reword the bullet points as follows:
"* *mboxNames* is an array of mbox locations for which the display notification will be sent to Target.
* *targetParameters* is the configured `ACPTargetParameters` for the request.